cmd/rpc.go

```diff
@@ -181,15 +190,15 @@ func (client *RPCClient) Call(serviceMethod string, args interface {
 }, reply interface{}) (err error) {
 	lockedCall := func() error {
 		client.RLock()
-		defer client.RUnlock()
-
 		if client.retryTicker != nil {
 			select {
 			case <-client.retryTicker.C:
 			default:
+				client.RUnlock()
 				return errRPCRetry
 			}
 		}
+		client.RUnlock()
```

Comment:
more readable:
```
        lockedCall := func() error {
                client.RLock()
                retryTicker := client.retryTicker
                client.RUnlock()
                if retryTicker != nil {
                        select {
                        case <-client.retryTicker.C:
                        default:
                                return errRPCRetry
                        }
                }

                // Make RPC call.
                args.SetAuthArgs(AuthArgs{client.authToken, client.args.RPCVersion, time.Now().UTC()})
                return client.rpcClient.Call(serviceMethod, args, reply)
        }
```

Reply:
Copying pointers doesn't make it protected. It is not good to copy it into new variables.. will keep the code as is.

Reply:
There was a typo, fixed it now.